src/config.js

```diff
@@ -35,6 +35,10 @@ export default {
       { name: 'Amazon Redshift', value: 'redshift' },
       { name: 'CockroachDB', value: 'cockroachdb' }
     ],
+    connectionMethods: [
```

Comment:
we could move this to the `saved_connection` and have a `supportsSocket` getter that checks if it is mysql, mariadb, or psql?